src/functionaltests/resources/InvalidSubscription.json

```diff
@@ -19,6 +19,7 @@
         "type": "ARTIFACT_1",
         "conditions": [
           {"jmespath": "gav.groupId=='com.mycompany.myproduct'"},
```

Comment:
Remove the jmespath condition containing gav.

Reply:
done